### ISSUE TITLE
Loss value keeps increasing in mobileNetV2 imagenet training 

### ISSUE THREAD

**Issue by jianyinglang** [tensorflow/models#9726]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/tree/master/research/slim

## 2. Describe the bug

The loss value in mobileNetV2 ImageNet training keeps increasing instead of decreasing.

## 3. Steps to reproduce

1. git clone the repo: https://github.com/tensorflow/models/tree/master/research/slim
2. Launch the mobileNetV2 run with the following script:
```
DATASET_DIR=$HOME/data/train
TRAIN_DIR=/tmp/train_logs
export NUM_GPUS=4
LR=0.045 * $NUM_GPUS
python3 train_image_classifier.py \
    --train_dir=${TRAIN_DIR} \
    --dataset_name=imagenet \
    --dataset_split_name=train \
    --dataset_dir=${DATASET_DIR} \
    --model_name="mobilenet_v2" \
    --max_number_of_steps=1000000 \
    --log_every_n_steps=10 \
    --batch_size=96 \
    --clone_on_cpu=False \
    --learning_rate=$LR \
    --preprocessing_name="inception_v2" \
    --label_smoothing=0.1 \
    --moving_average_decay=0.9999 \
    --num_clones=$NUM_GPUS \
    --learning_rate_decay_factor=0.98 \
    --num_epochs_per_decay=2.5/NUM_GPUS
```

## 4. Expected behavior

I expect the loss value drops after some steps.

## 5. Additional context

```
I0210 07:04:32.435021 139687965648704 learning.py:512] global step 510: loss = 42.0192 (0.500 sec/step)
INFO:tensorflow:global step 520: loss = 44.5649 (0.487 sec/step)
I0210 07:04:37.348536 139687965648704 learning.py:512] global step 520: loss = 44.5649 (0.487 sec/step)
INFO:tensorflow:global step 530: loss = 47.3959 (0.462 sec/step)
I0210 07:04:42.192924 139687965648704 learning.py:512] global step 530: loss = 47.3959 (0.462 sec/step)
INFO:tensorflow:global step 540: loss = 49.8404 (0.464 sec/step)
I0210 07:04:46.944735 139687965648704 learning.py:512] global step 540: loss = 49.8404 (0.464 sec/step)
INFO:tensorflow:global step 550: loss = 52.3198 (0.476 sec/step)
I0210 07:04:51.850367 139687965648704 learning.py:512] global step 550: loss = 52.3198 (0.476 sec/step)
INFO:tensorflow:global step 560: loss = 55.1327 (0.479 sec/step)
I0210 07:04:56.737862 139687965648704 learning.py:512] global step 560: loss = 55.1327 (0.479 sec/step)
INFO:tensorflow:global step 570: loss = 57.9392 (0.530 sec/step)
I0210 07:05:01.708699 139687965648704 learning.py:512] global step 570: loss = 57.9392 (0.530 sec/step)
INFO:tensorflow:global step 580: loss = 62.6942 (0.465 sec/step)
I0210 07:05:06.532044 139687965648704 learning.py:512] global step 580: loss = 62.6942 (0.465 sec/step)
INFO:tensorflow:global step 590: loss = 69.1686 (0.529 sec/step)
I0210 07:05:11.397589 139687965648704 learning.py:512] global step 590: loss = 69.1686 (0.529 sec/step)
INFO:tensorflow:global step 600: loss = 75.0341 (0.501 sec/step)
I0210 07:05:16.271065 139687965648704 learning.py:512] global step 600: loss = 75.0341 (0.501 sec/step)
INFO:tensorflow:global step 610: loss = 79.5997 (0.475 sec/step)
I0210 07:05:21.048773 139687965648704 learning.py:512] global step 610: loss = 79.5997 (0.475 sec/step)
INFO:tensorflow:global step 620: loss = 84.5228 (0.491 sec/step)
I0210 07:05:26.024133 139687965648704 learning.py:512] global step 620: loss = 84.5228 (0.491 sec/step)
INFO:tensorflow:global step 630: loss = 89.3062 (0.502 sec/step)
I0210 07:05:30.959553 139687965648704 learning.py:512] global step 630: loss = 89.3062 (0.502 sec/step)
INFO:tensorflow:global step 640: loss = 94.6541 (0.467 sec/step)
I0210 07:05:35.758488 139687965648704 learning.py:512] global step 640: loss = 94.6541 (0.467 sec/step)
INFO:tensorflow:global step 650: loss = 99.9105 (0.443 sec/step)
I0210 07:05:40.599656 139687965648704 learning.py:512] global step 650: loss = 99.9105 (0.443 sec/step)
INFO:tensorflow:global step 660: loss = 105.0693 (0.486 sec/step)
I0210 07:05:45.569366 139687965648704 learning.py:512] global step 660: loss = 105.0693 (0.486 sec/step)
INFO:tensorflow:global step 670: loss = 109.0619 (0.499 sec/step)
I0210 07:05:50.361270 139687965648704 learning.py:512] global step 670: loss = 109.0619 (0.499 sec/step)
INFO:tensorflow:global step 680: loss = 113.8180 (0.472 sec/step)
I0210 07:05:55.144923 139687965648704 learning.py:512] global step 680: loss = 113.8180 (0.472 sec/step)
INFO:tensorflow:global step 690: loss = 117.6049 (0.549 sec/step)
I0210 07:06:00.107023 139687965648704 learning.py:512] global step 690: loss = 117.6049 (0.549 sec/step
```

## 6. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 18.04
- Mobile device name if the issue happens on a mobile device: N/A
- TensorFlow installed from (source or binary): It is installed on AWS P3 tensorflow_p37
- TensorFlow version (use command below): 1.15.4
- Python version: Python 3.7.6
- Bazel version (if compiling from source): N/A
- GCC/Compiler version (if compiling from source): N/A
- CUDA/cuDNN version: gpu_cuda11.0
- GPU model and memory: Tesla V100 SXM2 16GB

**Comment by Hrayo712** [comment]
I have the same issue. Some help please :) ?

**Comment by abhishekbalu** [comment]
@jianyinglang  can you check you preprocessor name. I guess the preprocessor and the model have to be same. If you get an error change preprocessor to None in which case the model_name becomes the preprocessor name.

**Comment by jianyinglang** [comment]
Thanks @abhishekbalu! Would you mind elaborate what the preprocessor name is in mobileNetV2 pre-training?

**Comment by abhishekbalu** [comment]
@jianyinglang  the preprocessor name is always the model_name you use. In your case, it will also be mobileNetv2. for making it easy you can always keep it as None.  This will automatically set the preprocessor for you.

**Comment by saikumarchalla** [comment]
@jianyinglang  Is the issue resolved for you? If not, Please let us know. Thanks!

**Comment by saikumarchalla** [comment]
@jianyinglang  Closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!

**Comment by jianyinglang** [comment]
The issue seems caused by data processing. I used the data processed following https://sixninesit.com/training-a-resnet-50-imagenet-model-using-pytorch-on-a-single-aws-g4-or-p3-instance/ and got instability immediately. After changing a different dataset, the loss value does decay. However, I still do not get the target accuracy 72%.
